### DISCUSSION TITLE
Hi, is there a way to add query to http send out?

### DISCUSSION THREAD

**Discussion by kocoten1992** [php-webdriver/php-webdriver#1082] [General]
Hi, I've a situation, I have chromedriver running on a different machine than php-webdriver.

A is the machine have chromedriver and A have nginx proxy before chromedriver.

I call php-webdriver from B to A, but there is a problem.

I run multiple chromedriver instance on A, and I could config nginx to dynamically map a request to an internal chromedriver, for example:

```
    map $arg_port $proxy_port {
        default 4444;  # Default ChromeDriver port
        ~^[0-9]+$  $arg_port;  # Use the port number from the port argument
    }
```

I could call it like this: example.com?port=9515, but is there a way for me to modify out going http from php-webdriver? Specifically I'm looking for add `port=9515`.

**Comment by OndraM** [comment]
I'm not sure why and what you want to achieve, but this looks like usecase for [Selenium server grid](https://www.selenium.dev/documentation/grid/).

**Comment by kocoten1992** [comment]
thank you!
